### DISCUSSION TITLE
Распознавание диалектов

### DISCUSSION THREAD

**Discussion by rassie** [ilyhalight/voice-over-translation#254] [Общее]
Смотрю одно из свежих видео Тома Скотта про швейцарскую деревеньку (https://www.youtube.com/watch?v=R2oD1ZHNMFE). Ради интереса включаю перевод на немецкий. Перевод нормальный, неплохой. На отметке 1:00 начинается небольшой монолог женщины, члена местного управления. На видео сама она говорит на английском, перевод при этом переходит на чистейший швейцарский диалект немецкого! Самого Тома при этом переводят нормальным "немецким" немецким. 

Вот скажите мне кто-нибудь, как это сделано? На основе чего алгоритм переключился на швейцарский немецкий?

**Comment by NullVerdict** [comment]
> Смотрю одно из свежих видео Тома Скотта про швейцарскую деревеньку (https://www.youtube.com/watch?v=R2oD1ZHNMFE). Ради интереса включаю перевод на немецкий. Перевод нормальный, неплохой. На отметке 1:00 начинается небольшой монолог женщины, члена местного управления. На видео сама она говорит на английском, перевод при этом переходит на чистейший швейцарский диалект немецкого! Самого Тома при этом переводят нормальным "немецким" немецким.
> 
> Вот скажите мне кто-нибудь, как это сделано? На основе чего алгоритм переключился на швейцарский немецкий?

Я не могу дать точного ответа на этот вопрос, так как я не знаю, как работает алгоритм перевода Яндекса. 
Возможно, алгоритм определил, что женщина говорит на английском с швейцарским акцентом. Это могло повлиять на выбор диалекта для перевода. Также возможно, что алгоритм учитывает контекст видео. Если видео про Швейцарию и предназначено для немецкоговорящих зрителей, то использование швейцарского диалекта может быть более подходящим и приятным для них.
Если вас интересует это, вы можете обратиться к Яндексу напрямую, так как я не имею к нему отношения.

**Comment by ms-bubu** [comment]


**Reply by NullVerdict** [reply]
у тебя какие-то ошибки в консоли есть?
